collector_bmc_watchdog_native.go

```diff
@@ -0,0 +1,134 @@
+// Copyright 2021 The Prometheus Authors
```

Comment:
```suggestion
// Copyright 2025 The Prometheus Authors
```

Reply:
Should we maybe remove the date completely? From what I understand, it is not really necessary

Reply:
I don't know about "not necessary". But the standard for the Prometheus org is to set it to the year the file is created. This meets most of what we've read on various copyright annotation legal opinions.

Reply:
Fair enough. Updated all the new files' copyright years and also incorporated your other suggestion, thanks for reviewing!